src/js/mediasession.js

```diff
@@ -0,0 +1,132 @@
+import window from 'global/window';
+import {getMimetype} from './utils/mimetypes';
+
+/**
+ * @method initMediaSession Sets up media session if supported and configured
+ * @this { import('./player').default } Player
+ */
+export const initMediaSession = function() {
+  if (!this.options_.mediaSession || !('mediaSession' in window.navigator)) {
+    return;
+  }
+  const ms = window.navigator.mediaSession;
+  const defaultSkipTime = 15;
```

Comment:
Question, do you think it would be a good idea to use `skipButtons` values if they are available? This would allow a unified experience between the `player` and the `mediaSession`.

Reply:
I think that makes sense, yes.

Reply:
What do you think about keeping this property as it is for now and adding this modification via another PR? Because it would be nice to have the `titleBar` values as well.